app/user/rest.py

```diff
@@ -289,6 +290,7 @@ def send_user_sms_code(user_to_send_to, data):
 
     secret_code = create_secret_code()
     personalisation = {'verify_code': secret_code}
+    print(secret_code)
```

Comment:
debugging while writing the PR?

Reply:
Yes, interesting that tests didn't catch that 🤔